pino.js

```diff
@@ -338,6 +344,18 @@ function pino (opts, stream) {
     }
   }
 
+  var base = (typeof iopts.base === 'object') ? iopts.base : defaultBase
+
+  if (iopts.name !== undefined) {
+    base = Object.assign({}, base, {
+      name: iopts.name
+    })
+  }
+
+  if (base != null) {
```

Comment:
Is there a reason for the `!=` as apposed to `!==` or `(!base)`?

Reply:
no reason my mistake